packages/eslint-config-khan/package.json

```diff
@@ -1,32 +1,36 @@
 {
     "name": "@khanacademy/eslint-config",
-    "version": "5.2.1",
     "description": "ESLint config that applies Khan Academy's setings",
+    "author": "Kevin Barabash <kevinb@khanacademy.org>",
+    "license": "MIT",
+    "version": "5.2.1",
+    "publishConfig": {
+        "access": "public",
+        "provenance": true
+    },
     "repository": {
         "type": "git",
-        "url": "git@github.com:Khan/wonder-stuff.git",
+        "url": "https://github.com/Khan/wonder-stuff.git",
         "directory": "packages/eslint-config-khan"
     },
-    "publishConfig": {
-        "access": "public"
+    "bugs": {
+        "url": "https://github.com/Khan/wonder-blocks/issues"
     },
     "main": "index.js",
     "nobuild": true,
-    "author": "Kevin Barabash <kevinb@khanacademy.org>",
-    "license": "MIT",
```

Comment:
Key ordering across wonder-stuff packages varies... so many of these changes are just ordering the keys in a standard order.  These lines have moved up to line 4-5.